src/bindings/python/wheel/setup.py

```diff
@@ -315,7 +335,10 @@ def run(self):
         # remove symlink to avoid copying it, set RPATH
         self.post_install(LIB_INSTALL_CFG)
         # copy clib to package data (to WHEEL_LIBS_INSTALL_DIR)
-        self.copy_package_data(get_install_dirs_list(LIB_INSTALL_CFG))
+        self.copy_package_libs(get_install_dirs_list(LIB_INSTALL_CFG))
+        # copy package data (everything except python or libraries)
+        if CPACK_GENERATOR == "TGZ":
```

Comment:
I suppose every archive type is OK here. E.g. on Windows we use `ZIP` (see https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/features.cmake#L9)

Reply:
fixed